src/overlay/index.tsx

```diff
@@ -35,6 +35,7 @@ import { UpdateController } from '../modals/UpdateController';
 import { UpdatePayee } from '../modals/UpdatePayee';
 import { UpdateReserve } from '../modals/UpdateReserve';
 import { ValidatorMetrics } from '../modals/ValidatorMetrics';
+import { ValidatorDecentralization } from '../modals/ValidatorDecentralization';
```

Comment:
Again striving for a shorter name :)

```suggestion
import { ValidatorGeo } from '../modals/ValidatorGeo';
```

Reply:
I am happy to rename for making the code easier to read, shorter, etc. 

Geolocation is related to decentralization practically speaking.

However I have one reservation:

User facing I suggest we keep using the term "decentralization". For example in titles and help.

Polkawatch since the start is "decentralization analytics", and in many regards is a project designed to raise awareness among stakers about the need to "watch" the decentralization of our network. So, there is a lot of blog posts, documentation etc using this nomenclature. Dropping it completely would be counter productive.

Reply:
This has been applied, with comments.